### PR TITLE
Build or sanitycheck fixes for Openisa rv32m1/rv32m1_vega_ri5cy board

### PR DESCRIPTION
**Pull request by galak**
A few fixes to get the rv32m1_vega_ri5cy to pass sanitycheck completely.

### CODE REVIEW

**Review by mbolivar — APPROVED** [review]
Lgtm

File: scripts/sanitycheck [hunk 1]
```diff
@@ -869,7 +869,7 @@ class SizeCalculator:
                    'log_const_sections',"app_smem", 'shell_root_cmds_sections',
                    'log_const_sections',"app_smem", "font_entry_sections",
                    "priv_stacks_noinit", "_TEXT_SECTION_NAME_2",
-                   '_GCOV_BSS_SECTION_NAME', 'gcov', 'nocache']
+                   '_GCOV_BSS_SECTION_NAME', 'gcov', 'nocache', 'vectors']
```
Comment (MaureenHelm) [inline scripts/sanitycheck]: Should this be in ro_sections instead?

Comment (galak) [reply scripts/sanitycheck]: you'll have to ask @mbolivar on that

Comment (MaureenHelm) [reply scripts/sanitycheck]: Yes, it should be in ro_sections

Comment (galak) [reply scripts/sanitycheck]: Moved to ro_sections

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**